docs/PROFILING.md

```diff
@@ -59,56 +59,30 @@ go get github.com/uber/go-torch
 ```
 
 ## Generating a profile
-Before exploiting any result with go-torch and pprof we need to generate a profile - in our case a CPU profile. In this example we'll use the package `github.com/pkg/profile`.
-
-### Implement the code
-So on your main function start the profile:
-```go
-import "github.com/pkg/profile"
-
-var p interface {
-	Stop()
-}
-
-func main() {
-	p = profile.Start(profile.CPUProfile, profile.ProfilePath("."), profile.NoShutdownHook)
-    ...
-}
+Start snapd with the `--pprof` flag:
+```bash
+snapd -t 0 --pprof
 ```
 
-Then you must run `p.Stop()` just before exiting the program. In the snap daemon case it's just after handling a SIGINT (found in [snapd.go](../snapd.go)):
-```go
-func startInterruptHandling(modules ...coreModule) {
-    c := make(chan os.Signal, 1)
-    signal.Notify(c, os.Interrupt, os.Kill, syscall.SIGTERM)
-
-    //Let's block until someone tells us to quit
-    go func() {
-        sig := <-c
-        p.Stop()    // Stop profiling
-        ...
+You can now get at any time a profile during a defined period (e.g. 120 seconds):
+```bash
+curl "http://127.0.0.1:8181/v1/debug/pprof/profile?seconds=120" > cpu.pprof
```

Comment:
please mention that whatever time specified, the curl command needs to wait for the time interval.